### PR TITLE
Shorten most commonly used messages a bit

### PR DESCRIPTION
**Pull request by Marenthyu**
Signed-off-by: Marenthyu <marenthyu@marenthyu.de>

regarding #55

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -2162,12 +2162,12 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
 
                 if len(disenchants) == 1:
                     buytext = " (bounty filled)" if ordersFilled > 0 else ""
-                    self.message(channel, "Successfully disenchanted waifu %d%s and added %d points to %s's account" % (
+                    self.message(channel, "Successfully disenchanted waifu %d%s. Added %d points to %s's account" % (
```
Comment (xorhash) [inline main.py]: Idea for further shortening: `Successfully disenchanted waifu %d%s. %s gained %d points."

"gained points" is about half as long as "Added points to 's account".

File: main.py [hunk 2]
```diff
@@ -2178,7 +2178,7 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
             if command == "buy":
                 if len(args) != 1:
                     if len(args) > 0 and args[0].lower() == "booster":
-                        self.message(channel, "%s -> Did you mean !booster buy?" % tags['display-name'], isWhisper)
+                        self.message(channel, "%s, Did you mean !booster buy?" % tags['display-name'], isWhisper)
```
Comment (xorhash) [inline main.py]: Grammar: You should probably continue with lowercase after a comma.

File: main.py [hunk 3]
```diff
@@ -2387,14 +2387,14 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
...
-                        self.message(channel, "{user}, you open a {type} booster pack and you get: {droplink}".format(
+                        self.message(channel, "{user}, you open a {type} booster: {droplink}".format(
                             user=tags['display-name'], type=packname, droplink=droplink), isWhisper=isWhisper)
                     except InvalidBoosterException:
                         self.message(channel, "Invalid booster type. Packs available right now: %s." % visiblepacks,
                                      isWhisper=isWhisper)
                     except CantAffordBoosterException as exc:
                         self.message(channel,
-                                     "{user}, sorry, you don't have enough points to buy a {name} booster pack. You need {points}.".format(
+                                     "{user}, sorry, you don't have enough points to buy a {name} booster. You need {points}.".format(
```
Comment (xorhash) [inline main.py]: Idea for further shortening: `{name} pack` would save three characters over `{name} booster}`

**Comment by Marenthyu** [comment]
I was just thinking again about !checkhand - it would make it less intuitive, but i was thinking of maybe making it
```
xxx, your hand: 11(+1)/13
```
where 11 is held waifus, 1 current bounties (omitted if none), 13 hand slots

**Comment by Dabomstew** [comment]
Unless that saves a line on a usual-sized Twitch chat for an average-length username, it probably isn't worth doing.

**Comment by Marenthyu** [comment]
Consitering all the icons nepnepbot has, plus the timestamp, plus the custom ffz links, it would most likely save a line on my end, but i doubt that being possible for many others.
https://share.marenthyu.de/RTBjbQkv.png

**Comment by xorhash** [comment]
I was thinking about something like this when I went over the messages, but decided against it as-is. The bot is already not extremely friendly to new players (command line interfaces are pretty foreign to people), making the messages also hard to decipher won't help that. Checking your hand is an absolute core mechanic that's in my opinion too touchy to shorten like that.

**Pull request merged**